src/relentless/simulate/analyze.py

```diff
@@ -73,10 +73,15 @@ def _get_rdf_params(self, sim):
             else:
                 rdf_every = self.every
 
+            if "exclude" in self.rdf:
+                rdf_exclude = self.rdf["exclude"]
```

Comment:
Probably do like this to make sure it is cast to True/False
```suggestion
                rdf_exclude = bool(self.rdf["exclude"])
```